### PR TITLE
manage-stylesheets (Task No. 20)

### PR DESCRIPTION
**Pull request by netzarbeiter**

### CODE REVIEW

File: docs/manual/layout/theme-manager/manage-stylesheets.en.md [hunk 1]
```diff
@@ -55,56 +64,82 @@ The condition of a conditional comment can be read as follows:
...
 ### Class names of the Contao elements
 
-The CSS class names of the Contao elements are logical throughout. Content elements begin with the prefix`ce_` (for content element), followed by the type of the content element. For example, a text element has the class`ce_text`, a picture element has the class `ce_image`.
+The CSS class names of the Contao elements are logical throughout. Content elements begin with the prefix `ce_` (for 
+*c*ontent *e*lement), followed by the type of the content element. For example, a text element has the class `ce_text`, 
+a picture element has the class `ce_image`.
 
-The same applies to modules, except that they start with the prefix (for `mod_`modules). For example, the "Navigation Menu" module has the class`mod_navigation`, the "Message List" module the class`mod_newslist`. If you are not sure about the class of an element, just look in the source code of the web page.
+The same applies to modules, except that they start with the prefix `mod_` (for *mod*ules). For example, the 
+"Navigation Menu" module has the class `mod_navigation`, the "Message List" module the class `mod_newslist`. If you are
```
Comment (xchs) [inline docs/manual/layout/theme-manager/manage-stylesheets.en.md]:
```suggestion
"Navigation Menu" module has the class `mod_navigation`, the "News List" module the class `mod_newslist`. If you are 
```

File: docs/manual/layout/theme-manager/manage-stylesheets.en.md [hunk 2]
```diff
@@ -55,56 +64,82 @@ The condition of a conditional comment can be read as follows:
...
 
-The order of the format definitions plays an important role in Cascading Stylesheets, because each statement in a subsequent format definition can be overwritten. This feature is especially useful for compensating browser-specific peculiarities, for example:
+The order of the format definitions plays an important role in Cascading Stylesheets, because each statement in a 
+subsequent format definition can be overwritten. This feature is especially useful for compensating browser-specific 
+peculiarities, for example:
 
 ```css
-/* Außenabstand für alle Browser */
+/* Outer distance for all browsers */
 mod_search {
```
Comment (xchs) [inline docs/manual/layout/theme-manager/manage-stylesheets.en.md]:
```suggestion
.mod_search {
```

**Review by xchs — COMMENTED** [review]
👍🏻

**Comment by netzarbeiter** [comment]
Thank you, Hannes

**Pull request merged**